go/interpreter/lfvm/instructions_test.go

```diff
@@ -1380,6 +1380,69 @@ func TestInstructions_EIP2929_SSTOREReportsOutOfGas(t *testing.T) {
 	}
 }
 
+func TestInstructions_StorageOps_CallStorageContext(t *testing.T) {
+	address := tosca.Address{}
+	_, _ = rand.Read(address[:])
+	key := tosca.Key{}
+	_, _ = rand.Read(key[:])
+	value := tosca.Word{}
+	_, _ = rand.Read(value[:])
+
+	tests := map[OpCode]struct {
+		implementation func(*context) error
+		stack          []uint256.Int
+	}{
+		SLOAD: {
+			implementation: opSload,
+			stack: []uint256.Int{
+				*new(uint256.Int).SetBytes(key[:]),
+			},
+		},
+		SSTORE: {
+			implementation: opSstore,
+			stack: []uint256.Int{
+				*new(uint256.Int).SetBytes(key[:]),
+				*new(uint256.Int).SetBytes(value[:]),
+			},
+		},
+	}
+
+	for op, test := range tests {
+		t.Run(op.String(), func(t *testing.T) {
+			forEachRevision(t, op, func(t *testing.T, revision tosca.Revision) {
+
+				ctxt := getEmptyContext()
+				ctxt.params.Recipient = address
+				ctxt.params.Revision = revision
+				ctxt.stack = fillStack(test.stack...)
+				ctxt.gas = tosca.Gas(math.MaxInt64)
```

Comment:
does it need to be max gas for some reason ? considering this is this tests is not meant to test gas boundaries, I would rather simply see a big number such as 9999, or the addition of dynamic costs plus static cost.